examples/multigpu_advtrain/svhn_input.py

```diff
@@ -0,0 +1,85 @@
+# started from CIFAR10 in RevNets
```

Comment:
we already have a cleverhans/utils/utils_mnist.py, maybe this should be cleverhans/utils/utils_svhn.py

Reply:
Since this is a derivative work from cifar_input.py, it should probably have a copy of Mengye's license in it too

Reply:
The license fixed in 3c5de09. The file name fixed in 1fcb2a4.